Clients/GUI/Scripts/SignInner.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DTO;
+using HttpWebRequestWrapperLib;
+using System;
+
+namespace GUI.Scripts
+{
+    public class SignInner
+    {
+#pragma warning disable CS8632
+        public static string? SignIn(UserEmailPassword userInput)
```

Comment:
I want it to be void throwing exceptions, change my mind

Reply:
Fixed